packages/wallet/src/redux/protocols/direct-funding/__tests__/scenarios.ts

```diff
@@ -73,78 +51,68 @@ const bFundingReceivedEvent = globalActions.fundingReceivedEvent({
   amount: YOUR_DEPOSIT_B,
   totalForDestination: TOTAL_REQUIRED,
 });
-const postFundSetup0 = globalActions.commitmentReceived({
-  processId: channelId,
-  signedCommitment: scenarios.signedCommitment2,
-});
-const postFundSetup1 = globalActions.commitmentReceived({
-  processId: channelId,
-  signedCommitment: scenarios.signedCommitment3,
-});
+
+const sharedData = () => ({ ...advanceChannelScenarios.initialStore });
```

Comment:
Is there a reason we need this function?

Reply:
I'm not sure. We re-assign to `sharedData` at many points in reducers, so the value of `advanceChannelScenarios.initialStore` may change throughout a test suite.